code/game/machinery/computer/buildandrepair.dm

```diff
@@ -222,6 +222,11 @@
 	build_path = /obj/machinery/computer/card/minor/cmo
 	target_dept = TARGET_DEPT_MED
 
+/obj/item/circuitboard/card/minor/qm
+	name = "Circuit board (QM ID Computer)"
```

Comment:
Why not just do a ```board_name = "QM ID Computer"``` like the other ID boards?

Reply:
This'll be due to me having used an old attempt to make QM a head as a reference, seems I forgot to change this when I was making the first changes. This'll be updated in the next commit.